webpack.config.js

```diff
@@ -31,7 +31,13 @@ const config = [
     },
     devtool: (ENV !== 'production') ? 'source-map' : '',
     resolve: resolveJS(ENV, PATHS),
-    externals: externalJS(ENV, PATHS),
+    externals: Object.assign(
+      {},
+      externalJS(ENV, PATHS),
+      {
+        'state/tabs/ActionTabs' : 'ActionTabs'
+      }
+    ),
```

Comment:
Is this a patch for the admin PR before it's in webpack config?

Reply:
I added this to test the changes without having to rely on an PR to the webpack config repo just yet. I plan on removing this once the PR is not in WIP state any more.